scripts/prepare-machine.ps1

```diff
@@ -34,6 +34,16 @@ if ($IsWindows) {
         # TODO - Check for Windows preview
     }
 
+    if ($env:BUILD_BUILDNUMBER -ne $null) {
```

Comment:
Why do you have the if condition like this? I think you can just do `if ($Configuration -eq "Test")`

Reply:
Since this is modifying the global path and not the path just for the current instance of PS, it would be very easy for a user to accidentally override their configuration if it was just based off of test. In addition, we would need to add checks for if the configuration was ran multiple times. On Azure, those things are not an issue, so we can just globally set it using an azure specific variable.

We could add a user method for setting the path too, but I think we would put that under Dev, and add a bunch more checks to look up a few common locations.

Reply:
Either way, this didn't actually work on Azure. The variable is being set, but its not being propogated to other scripts. Hmm I'll try a few more things.

Reply:
So tried 3 different ways to set the PATH from the prepare-machines script and none of them will propagate. I posted on the issue yesterday that suggested `::add-path::` to see if thats some sort of hidden internal, since I've never seen that before.

Reply:
Tried a few more things, and couldn't solve the issue. However, then I thought since we have an azure specific check, we can just do the PATH update right before we call cdb in run-executable.ps1. It will only happen on Azure, and is the least invasive in terms of what needs to be modified.

Reply:
That's fine.